### PR TITLE
Use inlining annotations on important methods for all targets

### PR DESCRIPTION
**Pull request by mkustermann**

### CODE REVIEW

**Review by mraleph — APPROVED** [review]
LGTM with a comment

File: packages/flutter/lib/src/foundation/persistent_hash_map.dart [hunk 1]
```diff
@@ -384,17 +390,18 @@ List<Object?> _copy(List<Object?> array) {
...
+@pragma('vm:prefer-inline')
+@pragma('wasm:prefer-inline')
 List<Object?> _makeArray(int length) {
   return List<Object?>.filled(length, null);
 }
 
 /// This helper method becomes an no-op when compiled with dart2js on
 /// with high level of optimizations enabled.
 @pragma('dart2js:tryInline')
-@pragma('dart2js:as:trust')
```
[the remark this answers is not in the document]
Comment (mkustermann) [reply packages/flutter/lib/src/foundation/persistent_hash_map.dart:396]: Thanks. Done

**Comment by mkustermann** [comment]
test-exempt: non-semantic change

**Comment by matanlurey** [comment]
test-exempt: non-semantic changes

**Comment by mkustermann** [comment]
Thanks, @matanlurey !

**Pull request merged**